### PR TITLE
chore(ccm): make the naming style of ccm datasource unified

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:

make the naming style of ccm datasource unified

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

- This PR does not change the functional logic of the datasource.
- commit1: Rename the ccm datasource code file from `data_source_huaweicloud_scm_certificates` to `data_source_huaweicloud_ccm_certificates`.
- commit2: Discard the old documentation and provide a new documentation.
- commit3: Make function names and log messages more sensible.
- commit4: Add a new name pointing to the datasource.

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.

```
make testacc TEST='./huaweicloud/services/acceptance/ccm' TESTARGS='-run TestAccDatasourceCertificates_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ccm -v -run TestAccDatasourceCertificates_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceCertificates_basic
=== PAUSE TestAccDatasourceCertificates_basic
=== CONT  TestAccDatasourceCertificates_basic
--- PASS: TestAccDatasourceCertificates_basic (11.24s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ccm       11.291s
```

* [ ] Documentation updated.
* [ ] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/ccm/data_source_huaweicloud_ccm_certificates.go [hunk 1]
```diff
@@ -71,142 +60,121 @@ func certificatesCertificateSchema() *schema.Resource {
...
+	listResp, err := pagination.ListAllItems(
+		client,
 		"offset",
-		listCertificatesPath,
+		listPath,
 		&pagination.QueryOpts{MarkerField: ""})
 
 	if err != nil {
-		return common.CheckDeletedDiag(d, err, "error retrieving Certificates")
+		return diag.Errorf("error retrieving CCM SSL certificates.")
```
Comment (jinyangyang222) [inline huaweicloud/services/ccm/data_source_huaweicloud_ccm_certificates.go:154]: err should be return.

File: docs/data-sources/ccm_certificates.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  + **REVOKING**: The certificate revocation application is being reviewed.
+  + **REVOKED**: The certificate has been revoked.
+  + **UPLOAD**: The certificate is being managed.
+  + **CHECKING_ORG**: The organization verification is to be completed.
+  + **ISSUING**: The certificate is to be issued.
+  + **SUPPLEMENTCHECKING**: Additional domain names to be added for a multi-domain certificate are being reviewed.
+  
+  Defaults to **ALL**.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID to which the CCM certificate belongs.
```
Comment (jinyangyang222) [inline docs/data-sources/ccm_certificates.md:48]: Specifies the enterprise project ID to which the CCM certificates belong.

File: huaweicloud/services/ccm/data_source_huaweicloud_ccm_certificates.go (hunk 1 above)
Comment (deer-hang) [reply huaweicloud/services/ccm/data_source_huaweicloud_ccm_certificates.go]: OK

File: docs/data-sources/ccm_certificates.md (hunk 2 above)
Comment (deer-hang) [reply docs/data-sources/ccm_certificates.md]: OK

**Pull request merged**